CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+- **File Upload**:
+  - increased size limit for uploaded documents to 5 mb [#866](https://github.com/eclipse-tractusx/portal-frontend/pull/1187)
+
```

Comment:
Hi @ss-nikunj we have an continuous issue in the changelog file, the entry is at the wrong place and it's not even shown as conflict. I suggest you revert any change to the changelog and add the information which should be added to the changelog in the PR description and I add it then at the next release to the changelog file, sorry for the inconvenience.